tests/playwright/sanity/modules/checklist/selectors.ts

```diff
@@ -20,6 +20,8 @@ const selectors = {
 	progressBarPercentage: '#e-checklist header > div:nth-child(2) > div:nth-child(2)',
 	infotipFirstTimeClosed: '.e-checklist-infotip-first-time-closed',
 	infotipFirstTimeClosedButton: '.infotip-first-time-closed-button',
+	allDone: '.all_done',
+	gotItButton: '.all_done button',
```

Comment:
```suggestion
	allDone: '#e-checklist . MuiPaper-root . MuiPaper-root',
	gotItButton: '#e-checklist . MuiPaper-root . MuiPaper-root button',
```

Reply:
I think in this specific case it is justified, @ronenelementor @louiswol94 WDYT?

Reply:
I think it's ok. Maybe just use something like `e-checklist-done`